From 55b14ee8c61f28213f35188fb1410b39945759e3 Mon Sep 17 00:00:00 2001
From: Pardha Kesapragada <pkesapragada@habana.ai>
Date: Thu, 22 Apr 2021 08:37:21 +0300
Subject: [PATCH] [SW-0] gaudi2_arm: increase the ppboot size

Recent changes in the ppboot caused an
increase in size for ppboot image. Hence
increment the PPBOOT_SIZE to 20KB, just
like Greco legacy case.

Change-Id: I6f586e42fa1d50f4896b19762a2914333b8c2b84
Signed-off-by: Pardha Kesapragada <pkesapragada@habana.ai>
---

diff --git a/gaudi2_arm/Makefile b/gaudi2_arm/Makefile
index 56b8583..cf0367e 100644
--- a/gaudi2_arm/Makefile
+++ b/gaudi2_arm/Makefile
@@ -1,4 +1,4 @@
 FW_SIZE := 0x4400
-PPBOOT_MAX_SIZE := 16384
+PPBOOT_MAX_SIZE := 20480
 
-image_validation: image_validation_common
\ No newline at end of file
+image_validation: image_validation_common
diff --git a/gaudi2_arm/common.h b/gaudi2_arm/common.h
index 87a6e53..8ce8aea 100644
--- a/gaudi2_arm/common.h
+++ b/gaudi2_arm/common.h
@@ -13,7 +13,7 @@
 #define HL_FW_BOOT_PTRS_BASE		0x80000
 #define HL_SPI_LEGACY_PREBOOT_OFF	0xC0000
 
-#define PPBOOT_SIZE			0x4000		/* 16KB */
+#define PPBOOT_SIZE			0x5000		/* 20KB */
 #define PREBOOT_SIZE			0x10000		/* 64KB */
 #define SRAM_SIZE			0x3000000	/* 48MB */
 #define PPBOOT_TEXT_BASE		0x7FFD000000
